app.js

```diff
@@ -118,7 +118,7 @@ function isLoggedIn(req, res, next) {
   if (req.isAuthenticated())
     return next();
   // if they aren't redirect them to the home page
-  res.redirect('/login');
+  res.redirect('/api/login');
```

Comment:
We actually never end up using this middleware. We should keep it for when we actually start authenticating requests, just thought I'd mention it. For now, I'm going to change the redirect to just sending a `401` error.